pkg/detectors/databrickstoken/databrickstoken.go

```diff
@@ -45,19 +47,20 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 		resMatch := strings.TrimSpace(match[1])
 
 		for _, domainmatch := range domainMatches {
-			if len(domainmatch) != 2 {
-				continue
-			}
 			resDomainMatch := strings.TrimSpace(domainmatch[1])
 
 			s1 := detectors.Result{
 				DetectorType: detectorspb.DetectorType_DatabricksToken,
 				Raw:          []byte(resMatch),
-				RawV2:        []byte(resMatch + resDomainMatch),
+				RawV2:        []byte(resMatch + ":" + resDomainMatch),
```

Comment:
@ankushgoel27 This should remain the same

Reply:
please check my new commit with changes